src/wkwebview/mod.rs

```diff
@@ -1090,6 +1090,30 @@ r#"Object.defineProperty(window, 'ipc', {
     Ok(())
   }
 
+  pub fn bounds(&self) -> Rect {
+    if self.is_child {
```

Comment:
I believe we can run the same logic whether it is child or not or with a small tweak.